src/annotator/integrations/vitalsource.js

```diff
@@ -38,18 +39,14 @@ export function vitalSourceFrameRole(window_ = window) {
 }
 
 /**
- * Integration for the container frame in VitalSource's Bookshelf ebook reader.
- *
- * This frame cannot be annotated directly. This integration serves only to
- * load the client into the frame that contains the book content.
- *
- * @implements {Integration}
+ * Observe the book content iframe and load the client into this frame.
```

Comment:
This description could be clearer about where this code runs. eg. "VitalSourceInjector runs in the book container frame and loads the client into book content frames."